app/screens/settings/display_settings/display_settings.js

```diff
@@ -72,6 +90,19 @@ export default class DisplaySettings extends PureComponent {
             );
         }
 
+        const timezoneOption = (
```

Comment:
please make this setting available only if the server supports the timezone feature

Reply:
no need anymore, the timezone feature will be set by default